lib/parse/base.js

```diff
@@ -159,8 +159,6 @@ Base.prototype.getFileSource = function (filename) {
 			header: false,
 			bare: true
 		});
-	} else if (filename.match(this.jsxExtRegEx)) {
```

Comment:
To clarify, you're saying that the call to `jsx.transform` throws errors? Is that an issue with the `react-tools` library?